provider_base/templates/openvpn.json

```diff
@@ -0,0 +1,7 @@
+{
+  "openvpn": {
+    "gateway_address": "REQUIRED",
+    "ports": ["443"],
```

Comment:
the client does not support multiple ports, and the default openvpn port is often blocked by firewalls on public or corporate networks.